docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,177 @@
+
+## Detailed Guide
+
+### CPU
+
+The CPU will be responsible for many tasks, notably:
+
+- Transcoding Audio
+- Decoding video in unsupported codecs
+
+Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.
+
+:::caution
+
+Newer codecs can also require software decoding if there are no available hardware decoders. High bitrate HEVC content such as commonly found in 4k HDR can be very demanding even on modern CPUs. Please considering adding a modern GPU with 10bit HEVC decoding support.
+
+:::
+
+#### Integrated Graphics
+
+Integrated graphics can be useful for transcoding video. Please refer to the GPU section for a guide.
+
+#### Resizable BAR
+
+Resizable Bar (ReBAR) is a PCI Express feature that requires support from the platform. This feature is supported on Intel 10th gen or above and AMD Ryzen 3000 Series or above.
+
+When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.
+
+### System Memory (RAM)
+
+As a general rule of thumb, at least 8GB is recommended, though significantly smaller amounts may work just fine. If integrated graphics is being used, please consider adding more as integrated graphics usually use part of the system memory as video memory.
+
+### Graphics Cards (GPUs)
+
+With modern systems, the media engines are usually located on the GPUs. Therefore, the available hardware acceleration options are determined by the choice of GPU.
+
+Encoder Quality: Intel > Nvidia > AMD > Apple
+
+Intel graphics is recommended over Nvidia graphics because of the quality. AMD or Apple Silicon are not recommended since they produce poor quality H.264 and H.265(HEVC) output.
+
+:::note A Note on AMD AV1 Encoders
+
+With AV1, AMD has significantly improved the quality of their encoders. However, Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1. Even after Jellyfin adds support for transcoding to AV1, transcoding to H.264 or H.265(HEVC) is still more likely due to AV1 being in the early stages of adoption.
+
+:::
+
+A list of common codecs can be found [here](/docs/general/clients/codec-support/)
+
+The following is a list of codecs Jellyfin supports transocding to:
+
+- H.264 (Most common transcode target)
+- H.265 (Limited supported by clients)
+
+:::note AV1 Support
+
+Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1 as a target. Source content in AV1 will transcode to other codecs normally if needed. However, this might change in the future.
```

Comment:
The last sentences seems to reference the sentence before it but should the first sentence. The decoding capabilities will not change